.github/workflows/build.yml

```diff
@@ -56,43 +55,38 @@ jobs:
         run: |
           curl --continue-at - --create-dirs --location 'https://www.eclipse.org/downloads/download.php?file=/eclipse/downloads/drops4/R-${{ env.eclipse-version }}-${{ env.eclipse-date }}/eclipse-SDK-${{ env.eclipse-version }}-macosx-cocoa-aarch64.dmg&mirror_id=1' --output eclipse-sdk/eclipse.dmg
           hdiutil attach eclipse-sdk/eclipse.dmg
+          # Mac uses BSD cp which doesn't have --recursive therefore -R is required
           cp -r /Volumes/Eclipse/Eclipse.app /Applications/
           hdiutil detach /Volumes/Eclipse
           echo "eclipseRoot.dir=/Applications/Eclipse.app" | tee eclipsePlugin/local.properties
       # Disable gpg signing on *nix for time being until we get new keys established, checks will be on windows only for now
       #- name: Pre-build setup
-      #  if: matrix.os != 'windows-latest'
+      #  if: github.repository_owner == 'spotbugs' && matrix.os != 'windows-latest'
       #  run: |
       #    git fetch --no-tags https://$GITHUB_TOKEN@github.com/spotbugs/spotbugs.git +refs/heads/master:refs/remotes/origin/master
-      #    if [ "$GPG_SECRET_PASSPHRASE" != "" ]; then
-      #      gpg --quiet --batch --yes --decrypt --passphrase="$GPG_SECRET_PASSPHRASE" --output spotbugs.jks .github/workflows/spotbugs.jks.gpg;
-      #    fi
+      #    gpg --quiet --batch --yes --decrypt --passphrase="$GPG_SECRET_PASSPHRASE" --output spotbugs.jks .github/workflows/spotbugs.jks.gpg;
       #  env:
       #    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       #    GPG_SECRET_PASSPHRASE: ${{ secrets.GPG_SECRET_PASSPHRASE }}
       #    ORG_GRADLE_PROJECT_keystorepass: ${{ secrets.KEYSTORE_PASS }}
       - name: Pre-build setup Windows
-        if: matrix.os == 'windows-latest'
+        if: github.repository_owner == 'spotbugs' && matrix.os == 'windows-latest'
         run: |
           git fetch --no-tags https://$GITHUB_TOKEN@github.com/spotbugs/spotbugs.git +refs/heads/master:refs/remotes/origin/master
-          if ( "$GPG_SECRET_PASSPHRASE" -ne "" ) {
-            gpg --quiet --batch --yes --decrypt --passphrase="$GPG_SECRET_PASSPHRASE" --output spotbugs.jks .github/workflows/spotbugs.jks.gpg;
-          }
+          gpg --quiet --batch --yes --decrypt --passphrase="$GPG_SECRET_PASSPHRASE" --output spotbugs.jks .github/workflows/spotbugs.jks.gpg;
```

Comment:
It shouldn't be necessary to decrypt the .jks since it contained the certficate used to sign the Eclipse plugin

Reply:
yes I know, just cleaning this up.  build did fail, now I know why.  the gpg key @KengoTODA likely just zapped so its empty content now as this then ran like release which fails but has a subsequent call.  I'll comment out for now entirely though as we don't need this running at all currently as its not being used anywhere now.